devices/YarpRobotLoggerDevice/include/BipedalLocomotion/YarpRobotLoggerDevice.h

```diff
@@ -62,6 +64,8 @@ class YarpRobotLoggerDevice : public yarp::dev::DeviceDriver,
     std::unique_ptr<BipedalLocomotion::RobotInterface::YarpSensorBridge> m_robotSensorBridge;
     std::unique_ptr<BipedalLocomotion::RobotInterface::YarpCameraBridge> m_cameraBridge;
 
+    BipedalLocomotion::YarpUtilities::VectorsCollectionServer vectorCollectionRTDataServer;
```

Comment:
Can you use the ``m_``* pattern for private class members?

Reply:
Fixed in [3ee4481](https://github.com/nicktrem/bipedal-locomotion-framework/commit/3ee44813a2d3fff7918454e625dc4875650b3086)